src/pages/OrderListPage.tsx

```diff
@@ -0,0 +1,175 @@
+import { useState, useRef } from 'react';
+import { useLoaderData } from 'react-router-dom';
+import { Data } from 'types/types';
+
+import type { DatePickerProps } from 'antd';
+import { DatePicker } from 'antd';
+
+import { Button, Input, Space, Table, InputRef } from 'antd';
+import type { ColumnsType, ColumnType, TableProps } from 'antd/es/table';
+import type { FilterConfirmProps } from 'antd/es/table/interface';
+import { SearchOutlined } from '@ant-design/icons';
+import Highlighter from 'react-highlight-words';
+import dayjs from 'dayjs';
+
+const OrderListPage = () => {
+  const data = useLoaderData() as Data[];
+
+  const [searchText, setSearchText] = useState('');
+  const [searchedColumn, setSearchedColumn] = useState('');
+  const searchInput = useRef<InputRef>(null);
+
+  const [selectDate, setSelectDate] = useState('2023-03-08');
+
+  const selecetedData = data.filter(item => item.transaction_date === selectDate);
+
+  type DataIndex = keyof Data;
+
+  const handleSearch = (
+    selectedKeys: string[],
+    confirm: (param?: FilterConfirmProps) => void,
+    dataIndex: DataIndex,
+  ) => {
+    confirm();
+    setSearchText(selectedKeys[0]);
+    setSearchedColumn(dataIndex);
+  };
+
+  const handleReset = (clearFilters: () => void) => {
+    clearFilters();
+    setSearchText('');
+  };
+
+  const getColumnSearchProps = (dataIndex: DataIndex): ColumnType<Data> => ({
+    filterDropdown: ({ setSelectedKeys, selectedKeys, confirm, clearFilters, close }) => (
+      <div style={{ padding: 8 }} onKeyDown={e => e.stopPropagation()}>
+        <Input
+          ref={searchInput}
+          placeholder={`Search ${dataIndex}`}
+          value={selectedKeys[0]}
+          onChange={e => setSelectedKeys(e.target.value ? [e.target.value] : [])}
+          onPressEnter={() => handleSearch(selectedKeys as string[], confirm, dataIndex)}
+          style={{ marginBottom: 8, display: 'block' }}
+        />
+        <Space>
+          <Button
+            type='primary'
+            onClick={() => handleSearch(selectedKeys as string[], confirm, dataIndex)}
+            icon={<SearchOutlined />}
+            size='small'
+            style={{ width: 90 }}
+          >
+            Search
+          </Button>
+          <Button
+            onClick={() => clearFilters && handleReset(clearFilters)}
+            size='small'
+            style={{ width: 90 }}
+          >
+            Reset
+          </Button>
+          <Button
+            type='link'
+            size='small'
+            onClick={() => {
+              confirm({ closeDropdown: false });
+              setSearchText((selectedKeys as string[])[0]);
+              setSearchedColumn(dataIndex);
+            }}
+          >
+            Filter
+          </Button>
+          <Button
+            type='link'
+            size='small'
+            onClick={() => {
+              close();
+            }}
+          >
```

Comment:
이 부분은
```suggestion
            onClick={() => close}
          >
```

혹은 
```suggestion
            onClick={close}
          >
```
이렇게 수정해도 될 것 같습니다!